content/posts/google_sheets_as_apis.md

```diff
@@ -0,0 +1,148 @@
+
+   - For this tutorial, you don't need to grant any specific roles.
+   - Click **Continue**.
+
+4. **Grant Users Access to This Service Account**:
+
+   - Skip this step by clicking **Done**.
+
+5. **Generate a JSON Key File**:
+
+   - You should now see your service account listed.
+   - Click on the **Service Account** you just created.
+   - Navigate to the **Keys** tab.
+   - Click **Add Key** > **Create New Key**.
+   - Choose **JSON** as the key type.
+   - Click **Create**.
+   - A JSON key file will be downloaded to your computer (e.g., `service-account-key.json`).
+
+6. **Secure the Key File**:
+
+   - **Important**: Do not commit this file to version control. It's sensitive information.
+
+## Step 3: Share the Spreadsheet with the Service Account Email
+
+The Service Account acts like a virtual user. You need to share the spreadsheet with the Service Account's email address.
+
+1. **Get the Service Account Email**:
+
+   - In the **Details** tab of your Service Account, locate the **Email** field.
+   - It will look like: `your-service-account-name@your-project-id.iam.gserviceaccount.com`.
+
+2. **Share the Spreadsheet**:
+
+   - Open your Google Sheet in a browser.
+   - Click the **Share** button.
+   - Enter the Service Account email.
+   - Set the permission to **Editor** (or **Viewer** if you only need read access).
+   - Click **Send**.
+
+## Step 4: Create a new Genezio project
+
+We have prepared a couple of templates for you, depending what technology you want to use. Pick one of them, then click on the "Deploy now" button from one of the following Github repositories:
+
+- [Express.js - Google Spreadsheet App](https://github.com/Genez-io/google-sheets-as-apis-expressjs?tab=readme-ov-file#deploy)
+- [Genezio Typescript - Google Spreadsheet App](https://github.com/Genez-io/google-sheets-as-apis-typesafe?tab=readme-ov-file#deploy)
+
+## Step 5: Configure your project to access the spreadsheet data
+
+With the project deployed, go in the **Edit Code** interface in the browser and open the `server/service-account-key.json` file, and paste the contents of the `service-account-key.json` file you downloaded at Step 2.5.
+
+You will now need your Spreadsheet ID. Find it by:
+  - Open your Google Sheet.
+  - The URL will look like `https://docs.google.com/spreadsheets/d/SPREADSHEET_ID/edit#gid=0`.
+  - Copy the `SPREADSHEET_ID` part from the URL.
+
+Open the `server/backend.ts` file in the **Edit Code** interface and set the `SPREADSHEET_ID` variable to your Spreadsheet ID that you just copied.
+
+## Step 6: Test your project
+
+You can now re-deploy your project from the same **Edit Code** interface. Once the project is deployed, click on the **View App URLs** button to see the URL of your application. Open it to see your spreadsheet data in your app.
+
+## Step 7: See how this works
+
+Explore the `server/` and the `client/` folders to see how the data is being read and exposed in the application.
+
```

Comment:
I would add a conlcusion section:  

Conclusion
By following this guide, you can programmatically access Google Sheets data through Genezio using a Service Account, a secure and scalable solution for server-side applications. Whether you’re working with Node.js, Express.js, or Typescript, these steps provide a straightforward way to connect your app to Google Sheets.